### DISCUSSION TITLE
Saleor 3.2 Release

### DISCUSSION THREAD

**Discussion by karolkielecki** [saleor/saleor#9561] [Announcements]
Saleor 3.2 is now available! Introducing custom prices in checkout, customizable webhook payloads, public OIDC plugin, and more. Check the full changelog here: https://github.com/saleor/saleor/releases/tag/3.2.0

**Comment by jbergstroem** [comment]
Hello and thanks a lot for this release! 🎉 

Traditionally, Saleor seems to have release branches but in this case the `3.2` branch seems to lag behind the commit of the `3.2.0` tag. It would be great if the release branches could be kept up to date and tags linked ot the commit that updates said version in `pyproject.toml`, `package.json` and `CHANGELOG.md`.

- The 3.2.0 tag: https://github.com/saleor/saleor/tree/5763cb1ad22b84e29a270b9b0b365f32074b753d
- The 3.2 branch: https://github.com/saleor/saleor/tree/3.2

Finally, the `pyproject.toml` in the `main` branch seems to be at `3.1.0-a.0`.

**Reply by jbergstroem** [reply]
Note: the branch has since been updated! Thanks, @patrys.
